dev/tests/api-functional/testsuite/Magento/GraphQl/CatalogCustomer/PriceTiersTest.php

```diff
@@ -117,6 +117,41 @@ public function testGetLowestPriceForGuest()
         $this->assertEquals(round(7.25, 2), $this->getValueForQuantity(8, $itemTiers));
     }
 
+    /**
+     * @magentoApiDataFixture Magento/Catalog/_files/second_product_simple.php
+     * @magentoApiDataFixture Magento/Catalog/_files/three_simple_products_with_tier_price.php
+     */
+    public function testGetCorrectDisplaingTierPriceForProducts()
```

Comment:
There is a typo within test name, so let's adjust it into something like:
```suggestion
    public function testTierPricesAreCorrectlyDisplayedForProducts()
```